node/service/src/lib.rs

```diff
@@ -740,30 +851,35 @@ pub fn build_light(config: Configuration) -> Result<(TaskManager, RpcHandlers),
 pub fn build_full(
 	config: Configuration,
 	authority_discovery_disabled: bool,
+	is_collator: bool,
```

Comment:
Please make this an enum, as I have described it:

```
enum IsCollator {
    Yes,
    No,
}
```

Reply:
566fd216